### PR TITLE
Session cookie sameSite option

### LINKED ISSUE #2551: Cookie sameSite attribute
# Feature request

## Is your feature request related to a problem? Please describe.

Cannot auth from localhost because chrome blocks the cookie because its missing the samesite attribute

## Describe the solution you'd like

Add cookie samesite configuration to keystone constructor

### PR DESCRIPTION
**Pull request by timleslie**
This PR brings in the changes from #2602 with minor documentation tweaks. All credit goes to @ropaolle 🎉 

Fixes #2551

### CODE REVIEW

File: packages/keystone/README.md [hunk 1]
```diff
@@ -75,12 +73,31 @@ const keystone = new Keystone({
...
 
+#### Usage
+
+```javascript
+const keystone = new Keystone({
+  /* ...config */
+  cookie: {
+    secure = process.env.NODE_ENV === 'production', // Default to true in production
+    maxAge = 1000 * 60 * 60 * 24 * 30, // 30 days
+    sameSite = false,
```
Comment (Vultraz) [inline packages/keystone/README.md:96]: I think you accidentally swapped the code example and the `keystone` ctor destructuring default code 🤔

Comment (timleslie) [reply packages/keystone/README.md:96]: Oh, good catch 🤦

File: packages/keystone/README.md [hunk 2]
```diff
@@ -23,14 +23,12 @@ const keystone = new Keystone({
 | `adapter`        | `Object`   | Required                        | The database storage adapter. See the [Adapter framework](https://keystonejs.com/keystonejs/keystone/lib/adapters/) docs for more details.        |
 | `adapters`       | `Object`   | `undefined`                     | A list of named database adapters. Use the format `{ name: adapterObject }`.                                                                      |
 | `appVersion`     | `Object`   | See [`appVersion`](#appversion) | Configure the application version and where it is made available.                                                                                 |
-| `cookieMaxAge`   | `Int`      | 30 days                         | The maximum time, in milliseconds, session ID cookies remain valid.                                                                               |
-| `cookieSecret`   | `String`   | `qwerty`                        | The secret used to sign session ID cookies. Should be long and unguessable. Don't use this default in production!                                 |
```
Comment (Vultraz) [inline packages/keystone/README.md:27]: It looks like `cookieSecret` is sticking around but you removed it from the docs.

Comment (timleslie) [reply packages/keystone/README.md:27]: 😅 Nice catch, will fix.

**Comment by MadeByMike** [comment]
I like it

**Pull request merged**